package.json

```diff
@@ -13,11 +13,12 @@
     "setup": "yarn install",
     "postinstall": "simple-git-hooks",
     "lint:eslint": "eslint . --cache --ext js,jsx,ts,tsx",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' '**/*.html'",
+    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn '**/*.json' '**/*.md' '!**/CHANGELOG.md' '**/*.yml' '**/*.html'",
     "lint:changelogs": "yarn workspaces foreach --parallel --verbose run lint:changelog",
     "lint:tsconfig": "node scripts/verify-tsconfig.mjs",
-    "lint": "yarn lint:tsconfig && yarn lint:eslint && yarn lint:misc --check && yarn constraints",
-    "lint:fix": "yarn lint:tsconfig && yarn lint:eslint --fix && yarn lint:misc --write && yarn constraints --fix",
+    "lint": "yarn workspaces foreach --parallel run lint:eslint && yarn workspaces foreach --parallel run lint:misc --check && yarn lint:tsconfig && yarn constraints",
```

Comment:
Do we need to run prettier / misc in parallel? That seems fast without parallelizing due to the limited files in its scope.

Reply:
Is there a downside to doing so?

Reply:
Not sure its worth it with the overhead, it might actually be slower 😅

Reply:
This is not actually run in CI anyway, so I don't think it matters?